jenkins-master/jobs/scripts/workspace/config.py

```diff
@@ -31,10 +35,9 @@
                 'tier': 3,
             },
             'logs': {
-                # Currently we don't have a gecko.log because we log to the console (bug 1174766)
-                # 'gecko': os.path.join(here, 'build', 'upload', 'logs', 'gecko.log'),
-                'http.log': os.path.join(here, 'upload', 'logs', 'http.log'),
-                'tbpl.log': os.path.join(here, 'upload', 'logs', 'tbpl.log'),
+                'buildbot_text': os.path.join(here, 'build', 'upload', 'logs', 'log_info.log'),
+                'http.log': os.path.join(here, 'build', 'upload', 'logs', 'http.log'),
+                'report.html': os.path.join(here, 'build', 'upload', 'reports', 'report.html'),
```

Comment:
So, if I add a media type, and I have more files than these, I would list them similarly here?